src/library/TextInput/__tests__/TextInput.spec.js

```diff
@@ -1,12 +1,13 @@
 /* @flow */
 import React from 'react';
-import { shallow } from 'enzyme';
 import TextInput from '../TextInput';
 import examples from '../../../website/app/demos/TextInput/examples';
 import testDemoExamples from '../../../../utils/testDemoExamples';
+import { mountInThemeProvider } from '../../../../utils/enzymeUtils';
 
-function shallowTextInput() {
-  return shallow(<TextInput />);
+function mountTextInput(props = {}) {
+  const [, textInput] = mountInThemeProvider(<TextInput {...props} />);
```

Comment:
@brentertz For some reason, these tests fail if we don't mount the component. A shallow render is fine for the `TextInput `changes alone, but when we add the Pagination commits, mounting is necessary. Any ideas on why or whether there's a way to change it back?

Reply:
The shallow tests are very basic smoke tests.  They may not actually render html.  In this case it is likely due to the fact that you need a theme.  Perhaps you could render in a ThemeProvider and then use `.dive()`, but really, I'd just use mount.

...pushed a minor update